acceptance/bundle/sync/dryrun/script

```diff
@@ -0,0 +1,18 @@
+mkdir "project-folder" "ignored-folder" "ignored-folder/folder1"
+touch "project-folder/app.yaml" "project-folder/app.py" "project-folder/query.sql"
+touch "ignored-folder/script.py" "ignored-folder/folder1/script.py"
+cat > .gitignore << EOF
+ignored-folder/
+script
+output.txt
+repls.json
+EOF
+
+cleanup() {
+  rm .gitignore
+  rm -rf project-folder ignored-folder .git .databricks
+}
+trap cleanup EXIT
+
+# Note: output line starting with "Action: " lists files in non-deterministic order so we filter it out
```

Comment:
You're sorting anyway, so you already handle non-deterministic order, right?

> trace $CLI bundle sync --dry-run --output text | grep -v "^Action: " | sort

Reply:
I'm sorting output lines, but "Action:" is listing all files affected by the operation in a random order in a single line. I figured the easiest way around is to remove it from the output altogether

Reply:
Normally we try get rid of non-determinism in output (where feasible).

Is this due to map being used to hold the files? In that case I'd either get rid of map or use SortedKeys helper before printing.

Reply:
Addressed in #2824